models/intermediate/int_github__issue_joined.sql

```diff
@@ -59,7 +59,7 @@ pull_request as (
 
 select
   issue.*,
-  {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',"'/pull/'", 'issue.issue_number']) }} as url_link,
+  {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',if(issue.pull_request,"'/pull/'", "'/issues/'"), 'issue.issue_number']) }} as url_link,
```

Comment:
I wasn't able to get this sql to compile 🤔. It probably has something to do with the if statement within the jinja argument. I have seen these act up before.

It isn't as elegant, but I was able to get the code working with the suggested `case when` statement. Feel free to give it a test on your end and see if it works for you! I tested it on my side and saw the url be properly populated.
```suggestion
  case when issue.is_pull_request
    then {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',"'/pull/'", 'issue.issue_number']) }}
    else {{ dbt_utils.concat(["'https://github.com/'",'repository_teams.repository',"'/issues/'", 'issue.issue_number']) }}
      end as url_link,
```